packages/databricks-vscode/src/configuration/ConfigurationDataProvider.ts

```diff
@@ -272,16 +284,29 @@ export class ConfigurationDataProvider
             ];
 
             if (
-                syncDestination.remoteUri.name !== syncDestination.localUri.name
+                workspaceConfigs.syncDestinationType === "repo" &&
+                this.workspaceState.filesInWorkspaceFf &&
+                this.wsfsAccessVerifier.isEnabled
             ) {
                 children.push({
-                    label: "The remote sync destination name does not match the current vscode workspace name",
-                    tooltip:
-                        "If syncing to directory with a different name is the intended behaviour, this warning can be ignored",
+                    label: {
+                        highlights: [[0, 10]],
+                        label: "Deprecated",
```

Comment:
I wonder if we can find a better term than `Deprecated`. I also wonder how many users have to stay on repos

Reply:
Users will only see deprecated if they are on a cluster with dbr >= 11.2